From: Jakub Jirutka <jakub@jirutka.cz>
Date: Mon, 3 Apr 2017 19:39:00 +0200
Subject: [PATCH] Fix on newer Closure Compiler, change language_in to ES6

And also fix name of the compilation_level.

Upstream-Issue: https://github.com/kripken/emscripten/issues/5109

--- a/tools/shared.py
+++ b/tools/shared.py
@@ -2140,8 +2140,8 @@
       args = [JAVA,
               '-Xmx' + (os.environ.get('JAVA_HEAP_SIZE') or '1024m'), # if you need a larger Java heap, use this environment variable
               '-jar', CLOSURE_COMPILER,
-              '--compilation_level', 'ADVANCED_OPTIMIZATIONS',
-              '--language_in', 'ECMASCRIPT5',
+              '--compilation_level', 'ADVANCED',
+              '--language_in', 'ECMASCRIPT6',
               '--externs', CLOSURE_EXTERNS,
               #'--variable_map_output_file', filename + '.vars',
               '--js', filename, '--js_output_file', filename + '.cc.js']
